rambo/vagrant_resources/vagrant/core

```diff
@@ -52,3 +52,51 @@ Vagrant.configure("2") do |config|
     puts "Vagrant done with destroy."
   end
 end
+
+#redundant code from LXC and VirtualBox vagrant files
```

Comment:
I'd like this removed. After this change, it will no longer be redundant, simply reused like the rest of this file.